### PR TITLE
Improve Ableton Live Casks

### PR DESCRIPTION
**Pull request by danielbayley**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/ableton-live-suite.rb [hunk 1]
```diff
@@ -1,14 +1,28 @@
 cask 'ableton-live-suite' do
-  version '10.1.4'
-  sha256 '8962eab69f8600501d5002b051668ea1ddde3c3febb44d02253da5064364a62b'
+  version '10.1.1'
```
Comment (vitorgalvao) [inline Casks/ableton-live-suite.rb]: Why the downgrade?

**Comment by vitorgalvao** [comment]
Please, one PR per cask. More often than not, PRs with multiple casks have a tiny mistake that holds up every cask. In addition, CI doesn’t run with more than one cask.

File: Casks/ableton-live-suite.rb (hunk 1 above)
Comment (danielbayley) [reply Casks/ableton-live-suite.rb]: @vitorgalvao Not sure what happened there… must be something to do with the fact that the suite version is the one I use and have installed. Maybe it wasn't up to date when I did this…

**Comment by danielbayley** [comment]
> CI doesn’t run with more than one cask

@vitorgalvao I wonder if it should be able to, for versions of the same cask…?

**Comment by vitorgalvao** [comment]
> I wonder if it should be able to, for versions of the same cask…?

Too much work and code (and thus bugs) for something that doesn’t happen often enough and isn’t a big deal to do manually.

**Pull request closed without merging**